pkg/storage/tpr/fake_core_rest_client_test.go

```diff
@@ -191,33 +188,36 @@ func getItems(storage namespacedStorage) func(http.ResponseWriter, *http.Request
 			if !ok {
 				log.Fatalf("error performing type assertion: %s", err)
 			}
-			accessor.SetKind(item, "")
```

Comment:
This fake got copied over from the one we were (at one time) using in integration tests. This line probably didn't belong and reflects a time when we didn't understand things as well. Ask instead why it _should_ be there. It shouldn't. 😄